### ISSUE TITLE
Player spawning

### ISSUE THREAD

**Issue by JuiceGrape** [JuiceGrape/Incarnation-Of-Ragnarok#23]
Blocked by #4, #7, and #25 

When the player enters a dungeon, it will load up a random room from the list of possible starting rooms. These rooms will be special rooms, with an larger amount of exits for the player to choose from. When the player enters this room, their saved equipment should attach to them, the proper animation controller should be attached, and any possible buffs / debuffs from gear should be applied. The scene should fade in and out as it does with normal rooms, and player control should not start until everything is ready.

Out of scope:
- Dungeon entering (loading a random room).
- Attaching gear models to the player based on equipped gear
- Changing player model based on equipped gear
